presto-spi/src/main/java/com/facebook/presto/spi/CatalogSchemaTableName.java

```diff
@@ -39,12 +36,17 @@ public static CatalogSchemaTableName valueOf(String catalogSchemaTableName)
 
     public CatalogSchemaTableName(String catalogName, SchemaTableName schemaTableName)
     {
+        checkNotEmpty(catalogName, "catalogName");
```

Comment:
Both of these methods return the value, so the check and assignment can be done together. This makes the code more concise and makes it easier to see that all assignments are checked.
```java
this.catalogName = checkNotEmpty(catalogName, "catalogName");